hnn_core/network.py

```diff
@@ -1374,12 +1374,17 @@ def clear_connectivity(self):
 
     def clear_drives(self):
         """Remove all drives defined in Network.connectivity"""
-        connectivity = list()
-        for conn in self.connectivity:
-            if conn['src_type'] not in self.external_drives.keys():
-                connectivity.append(conn)
+        self.connectivity = [conn for conn in self.connectivity if
+                             conn['src_type'] if conn['src_type'] not
+                             in self.external_drives.keys()]
+
+        for cell_type in list(self.gid_ranges.keys()):
```

Comment:
Probably a matter of semantics but are drive names also "cell types"? I think in other parts of the codebase "cell_types" usually just refers to the neocortical cell names? At least the Network["cell_types"] is only those cells. Maybe something like "range_name" would be clearer.

Reply:
yeah we have a bit of inconsistency here ... how about calling it `cell_name` like we do in the rest of the repo. The variable name `cell_type` and `cell_template` are a bit interchangeable and confusing